### ISSUE TITLE
Revising the default http://cdn.opensuse.org

### ISSUE THREAD

**Issue by lkocman** [openSUSE/openSUSE-repos#72]
@bmwiedemann @dirkmueller is it the start shall we start using https://cdn.opensuse.org or https://download.opensuse.org?
I recall Bernhard mentioning that download-o-o will redirect you to cdn-o-o too.\

Thank you

**Comment by bmwiedemann** [comment]
IMHO, it is better to use download.o.o - maybe in the http variant to have more mirrors available?

1. it will not use the CDN for critical repo + key files for consistency+security
2. it will handle European users directly for lower latency
3. it will redirect other users+files to the CDN

**Comment by dirkmueller** [comment]
> IMHO, it is better to use download.o.o - maybe in the http variant to have more mirrors available?
> 
> 1. it will not use the CDN for critical repo + key files for consistency+security

That's not true. the cdn redirect happens on a hostname basis, not on a file/repo level. 

> 2. it will handle European users directly for lower latency

Again, not true. download.o.rg is slower than cdn.o.org for files that are in its cache.

> 3. it will redirect other users+files to the CDN

true, but if 1. and 2. would be true then 3. would be kinda an argument against rather than for the change.

**Comment by dirkmueller** [comment]
I agree that openSUSE-repos shouldn't have a different hostname setting than the embedded product repos (which today are download.o.org in http variant). these should be always the same imho. 

the real question is whether we want to split the default from one to the other. I don't think we want that, because the geoip redirect feature of d.o.org helps us to more dynamically switch the workload to and away from the cdn.

**Comment by bmwiedemann** [comment]
> >  it will not use the CDN for critical repo + key files for consistency+security

> That's not true. the cdn redirect happens on a hostname basis, not on a file/repo level.

https://github.com/openSUSE/libzypp/pull/422/files#diff-c0032603730799a899e5cf052af4c41c2272b28e1c4d1cade7e081f6b84456aaR270 says
> Some specific files are however excluded from this redirection due to security reasons, especially the repo metdata index and it's key and checksum files: repomd.xml, repomd.xml.key and repomd.xml.asc

**Comment by dirkmueller** [comment]
> Some specific files are however excluded from this redirection due to security reasons, especially the repo metdata index and it's key and checksum files: repomd.xml, repomd.xml.key and repomd.xml.asc

oh wow, what a hack :( this is really bad though for performance. those files are very high latency.

**Comment by dirkmueller** [comment]
given this hack, I change my opinion: the default should be cdn.o.org then globally.
